### DISCUSSION TITLE
play image list

### DISCUSSION THREAD

**Discussion by ymmmvp36** [alexmercerind/dart_vlc#173] [Q&A] [answered]
i use this to play image and video.when the source is image type,it have the default duration time for 10s . i want to change this default duration time.but i haven't found the right way,can u tell me the right way? thank you very much.

**Comment by alexmercerind** [comment, marked as answer]
I never played images using VLC / libVLC. But you can use `--image-duration` argument to set duration (see more [here](https://wiki.videolan.org/VLC_command-line_help/)).
None the less, not really difficult to implement in Flutter itself.

**Reply by ymmmvp36** [reply]
thank u
